db/migrate/20200913140801_remove_default_from_racer_gender.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDefaultFromRacerGender < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default :racers, :gender, from: 0, to: nil
```

Comment:
The `from:` and `to:` syntax makes this migration reversible.

Reply:
Great call on this PR.